server/src/main/scala/passengersplits/core/PassengerInfoRouterActor.scala

```diff
@@ -122,6 +123,13 @@ class AdvancePassengerInfoActor extends PersistentActor with PassengerQueueCalcu
       state.copy(latestFileName = Option(zipFilename))
       completionMonitor ! VoyageManifestZipFileCompleteAck(zipFilename)
 
+    case FlushOldVoyageManifests(before) =>
+      state.copy(flightManifests = state.flightManifests.filterNot {
+        case (_, vm) if vm.scheduleArrivalDateTime.map(_.millisSinceEpoch).getOrElse(0L) < before.millisSinceEpoch =>
+          log.info(s"Dropping voyage manifest ${vm.summary}")
```

Comment:
add to log message the scheduled time and the cut off time for clarity of the decision to drop